addons/purchase_requisition/migrations/10.0.0.1/post-migration.py

```diff
@@ -8,16 +8,19 @@
 
 def fill_product(env):
     """Fill with an pre-created product as it's required."""
-    product = env['product.product'].create({
-        'active': False,
-        'name': 'OU purchase requisition line wildcard',
-    })
-    openupgrade.logged_query(
-        env.cr, """
-        UPDATE purchase_requisition_line
-        SET product_id = %s
-        WHERE product_id IS NULL;""", (product.id, ),
-    )
+    if env['purchase.requisition.line'].search_count(
+            [('product_id', '=', False)]) > 0:
+        product = env['product.product'].create({
+            'active': False,
+            'categ_id': 1,  # categ_id is required
+            'name': 'OU purchase requisition line wildcard',
```

Comment:
Thanks for fixing flake8! Can you insert an additional fix for me? When creating utility products in other cases, we found out that we need to set 'company_id' to False to cover multicompany contexts. Can you do that here?